#!/usr/bin/env python
PACKAGE = "mav_disturbance_observer"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("q_position",    double_t,    0, "position noise", 0.1 ,  0.001, 10.0)
gen.add("q_velocity",    double_t,    0, "velocity noise", 0.1 ,  0.001, 10.0)

gen.add("r_position",    double_t,    0, "position meas noise", 0.1 ,  0.001, 10.0)
gen.add("r_velocity",    double_t,    0, "velocity meas noise", 0.1 ,  0.001, 10.0)


exit(gen.generate(PACKAGE, "Config", "KFPredictionObserver"))